### PR TITLE
DOCSUP-16607: Update the SQL-statement CREATE DICTIONARY 

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #29899

### CODE REVIEW

File: docs/en/operations/system-tables/dictionaries.md [hunk 1]
```diff
@@ -15,9 +16,11 @@ Columns:
     -   `FAILED_AND_RELOADING` — Could not load the dictionary as a result of an error and is loading now.
 -   `origin` ([String](../../sql-reference/data-types/string.md)) — Path to the configuration file that describes the dictionary.
 -   `type` ([String](../../sql-reference/data-types/string.md)) — Type of a dictionary allocation. [Storing Dictionaries in Memory](../../sql-reference/dictionaries/external-dictionaries/external-dicts-dict-layout.md).
--   `key` — [Key type](../../sql-reference/dictionaries/external-dictionaries/external-dicts-dict-structure.md#ext_dict_structure-key): Numeric Key ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) or Сomposite key ([String](../../sql-reference/data-types/string.md)) — form “(type 1, type 2, …, type n)”.
+-   `key` — [Key type](../../sql-reference/dictionaries/external-dictionaries/external-dicts-dict-structure.md#ext_dict_structure-key): Numeric Key ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) or Сomposite key ([String](../../sql-reference/data-types/string.md)) — form "(type 1, type 2, …, type n)".
```
Comment (olgarev) [inline docs/en/operations/system-tables/dictionaries.md:19]:
```suggestion
-   `key` ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) or [String](../../sql-reference/data-types/string.md)) — Numeric or composite [key](../../sql-reference/dictionaries/external-dictionaries/external-dicts-dict-structure.md#ext_dict_structure-key) (composite key is a list like `(type 1, type 2, …, type n)`).
```

Comment (olgarev) [inline docs/en/operations/system-tables/dictionaries.md:19]:
```suggestion
-   `key` — [Key type](../../sql-reference/dictionaries/external-dictionaries/external-dicts-dict-structure.md#ext_dict_structure-key): Numeric Key ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) or Сomposite key ([String](../../sql-reference/data-types/string.md)) — form `(type 1, type 2, …, type n)`.
```

Comment (olgarev) [inline docs/en/operations/system-tables/dictionaries.md:19]:
```suggestion
```

**Comment by sevirov** [comment]
@kitaisreal Please, review this PR.

**Review by kitaisreal — APPROVED** [review]

**Pull request merged**